packages/example/src/components/remote-control/OverridableEventEmitter.ts

```diff
@@ -0,0 +1,25 @@
+import { SupportedKeys } from './SupportedKeys';
+
+class OverridableEventEmitter {
+  private handlers = new Map<string, ((event: SupportedKeys) => boolean)[]>();
+
+  on = (eventName: string, handler: (event: SupportedKeys) => boolean) => {
+    if (!Array.isArray(this.handlers[eventName])) this.handlers[eventName] = [];
+    this.handlers[eventName].push(handler);
+  };
+
+  off = (eventName: string, handler: (event: SupportedKeys) => boolean) => {
+    this.handlers[eventName] = this.handlers[eventName].filter((h) => h !== handler);
+  };
+
+  emit = (eventName: string, event: SupportedKeys) => {
+    for (let index = this.handlers[eventName].length - 1; index >= 0; index--) {
+      const handler = this.handlers[eventName][index];
+      if (handler(event)) {
+        return;
+      }
+    }
+  };
+}
+
+export default new OverridableEventEmitter();
```

Comment:
you're exposing a singleton : you should only expose the class, and instantiate it in the RemoteControlManager ;) (like we used to do with mitt)